testbot_description/launch/displayHuman.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+<group ns="human">
+<arg name="model" />
+<arg name="gui" default="False" />
+<param name="robot_description" textfile="$(arg model)"  />
+<param name="use_gui" value="$(arg gui)"/>
+<node name="state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+</group>
+</launch>
+
```

Comment:
The previous comment applies to all non-indented xml files. 
Please note that consistent indentation is a tool that is typically easy to automatize, so you should not do it manually. Search in Google "automatic xml indentation" or something similar to check existing tools doing this job.